modules/packages/LinearAlgebra.chpl

```diff
@@ -188,6 +188,7 @@ module LinearAlgebra {
 
 import BLAS;
 use LAPACK only lapack_memory_order, isLAPACKType;
+use Math;
```

Comment:
I thought `Math` was automatically `use`d everywhere. @lydia-duncan, @e-kayrakli -- should we be including this here?

Reply:
I believe Nikhil is right and the use of Math is unnecessary.  Did you encounter issues without it?

Reply:
It works without math. I believe I incorporated it because I needed access to some function which I no longer need now.